### PR TITLE
Add line breaks around code blocks to .eslinrc

### PR DESCRIPTION
**Pull request by maceonthompson**

### CODE REVIEW

**Review by achillesecos — APPROVED** [review]
LGTM

**Review by FabriceCastel — APPROVED** [review]
LGTM

File: frontend/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint-fix": "eslint --fix ./"
+    "lint-fix": "eslint --fix ./src/"
```
Comment (kristen-law) [inline frontend/package.json:28]: Why was this changed to the src/ directory? It looks like most of the javascript code is under the frontend/ directory
